### ISSUE TITLE
Inconsistent use of the term "hidden"

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/aria#1159]
The aria-hidden description says the following:

> source: https://www.w3.org/TR/wai-aria-1.2/#aria-hidden
> An element is considered hidden if it, or any of its ancestors are not rendered or have their aria-hidden attribute value set to true.

This second part, of an element being considered hidden when it has aria-hidden="true" is inconsistent with the definition of hidden, which says: 

> Indicates that the element is not visible, perceivable, or interactive to any user. An element is considered hidden if it or any one of its ancestor elements is not rendered or is explicitly hidden.

"not visible ... to any user" is not necessarily the case for elements with aria-hidden="true". More commonly, aria-hidden="true" is used for elements that are visible but should not be included in the accessibility tree.

**Comment by JAWS-test** [comment]
The clause "or have their aria-hidden attribute value set to true" should be removed

**Comment by jnurthen** [comment]
@JAWS-test I don't think that is the right way to resolve - although I agree there is an issue. I think the best way to resolve might be to modify the hidden definition although that definition also gets pulled into accname and core-aam so will need to evaluate the consequences of changing it.

**Comment by jnurthen** [comment]
Looking at the other documents the usages of hidden include any uses where the element is hidden using aria-hidden so changing the definition to allow this usage seems like it would resolve issues in multiple specs.

**Comment by JAWS-test** [comment]
@jnurthen Of course the definition of `hidden` can also be changed, but I would like to point out that it would then have to be changed in all specifications, because e.g. [Core AAM](https://www.w3.org/TR/core-aam-1.2/#dfn-hidden) also says:

> Indicates that the element is not visible, perceivable, **or** interactive to any user. 

I wonder if the "or" is correct. It should be "and", right? Otherwise, "hidden" would also be: visible, but not operable

**Comment by jnurthen** [comment]
@JAWS-test yes I understand that and said the same in my previous 2 comments. Any term in the important terms section is pulled in as needed from a central location based on usage of that term in the document. I took an initial look at core-aam and accname and believe that hidden in these documents is meant to include aria-hidden content whether it is visible or not. Hence the definition should be changed.

**Comment by sinabahram** [comment]
Listing documents and objectives below to make sure I have a proper understanding from the call.

####Documents
1. [Core-AAM](https://www.w3.org/TR/core-aam-1.2)
2. [AccName](https://github.com/w3c/accname)
3. [ARIA](https://github.com/w3c/aria)

####Objectives:
1. Determine that all instances of hidden are consistent, and reaching out on here when they are not to identify how to resolve.
2. Update definition of hidden in glossary.

**Comment by jnurthen** [comment]
Not a new issue in 1.2 - moving to 1.3

**Comment by WilcoFiers** [comment]
No objection to a 1.3 fix. This is a pretty minor point. I think it's clear what is meant here.

**Comment by scottaohara** [comment]
assigning myself so that i can have a reminder to help out if i can.
